{"name": "CVE-2016-7952", "public_date": "2016-09-25T00:00:00", "package_state": [{"fix_state": "Will not fix", "product_name": "Red Hat Enterprise Linux 5", "package_name": "libXtst", "cpe": "cpe:/o:redhat:enterprise_linux:5"}, {"fix_state": "Will not fix", "product_name": "Red Hat Enterprise Linux 6", "package_name": "libXtst", "cpe": "cpe:/o:redhat:enterprise_linux:6"}, {"fix_state": "Will not fix", "product_name": "Red Hat Enterprise Linux 7", "package_name": "libXtst", "cpe": "cpe:/o:redhat:enterprise_linux:7"}], "document_distribution": "Copyright \u00a9 2016 Red Hat, Inc. All rights reserved.", "cvss3": {"status": "draft", "cvss3_scoring_vector": "CVSS:3.0/AV:A/AC:H/PR:N/UI:N/S:U/C:L/I:L/A:L", "cvss3_base_score": "5.0"}, "threat_severity": "Moderate", "bugzilla": {"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1381922", "description": "\nCVE-2016-7952 libXtst: Insufficient validation of server responses result in various data mishandlings\n    ", "id": "1381922"}, "references": ["\nhttps://lists.x.org/archives/xorg-announce/2016-October/002720.html\n    "], "details": ["\nX.org libXtst before 1.2.3 allows remote X servers to cause a denial of service (infinite loop) via a reply in the (1) XRecordStartOfData, (2) XRecordEndOfData, or (3) XRecordClientDied category without a client sequence and with attached data.\n    "], "upstream_fix": "libXtst 1.2.3", "cvss": {"cvss_scoring_vector": "AV:A/AC:M/Au:N/C:P/I:P/A:P", "status": "draft", "cvss_base_score": "5.4"}}